src/sophys/common/utils/signals.py

```diff
@@ -191,6 +192,31 @@ def __init__(self, prefix, **kwargs):
         super().__init__(prefix + "-Mon", **kwargs)
 
 
+class EpicsSignalWithRetryRO(EpicsSignalRO):
```

Comment:
This can easily be generalized to have an argument in the ctor for the number of retries. I don't see why not do that right now.